utils/reputation/client.go

```diff
@@ -130,18 +136,23 @@ func (c *HTTPClient) IsWalletReputable(
 ) (bool, error) {
 	req, err := c.newRequest(
 		ctx,
-		"GET", "v1/reputation/"+paymentID.String(),
+		"GET",
+		"v1/reputation/"+paymentID.String(),
 		nil,
 	)
 	if err != nil {
-		return false, err
+		return false, errs.New(0, "request", nil)
```

Comment:
it seems strange to shuttle this error into a struct called `ResponseErrorBundle`, i would only use that after the request is attempted